server/controllers/user.js

```diff
@@ -10,11 +10,11 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
   let users;
   if (searchString) {
     users = await User.find({
-      username: { $regex: searchString, $options: "i" }
+      username: { $regex: searchString, $options: "i" },
     });
   }
 
-  if (!users) {
+  if (users.length === 0) {
     res.status(404);
     throw new Error("No users found in search");
   }
```

Comment:
Could we actually remove this route and controller. We aren't going to have functionality for searching for users.